### PR TITLE
[NewUI] Clear send state on cancelling and signing.

### PR DESCRIPTION
**Pull request by danjm**
Ensures send values stored in redux state are cleared when user leaves the send screen.

### CODE REVIEW

File: ui/app/send-v2.js [hunk 1]
```diff
@@ -445,6 +449,8 @@ SendTransactionScreen.prototype.onSubmit = function (event) {
     txParams.to = to
   }
 
+  clearSend()
```
Comment (sdtsui) [inline ui/app/send-v2.js]: Why is this in `onSubmit`?  If we ever need to reuse state between submits (error handling maybe), would this be limiting?  I'm thinking of how "when the user leaves a screen" is often synonymous with "stuff that happens on lifecycle/unmount.

**Review by sdtsui — COMMENTED** [review]
@chikeichan merged this as I was reviewing - gonna send you the link on Slack. Minor comment.

File: ui/app/send-v2.js (hunk 1 above)
Comment (danjm) [reply ui/app/send-v2.js]: Good point. One option would be to clear it on mount. Then we will always have the last sent transactions data in state up until the point when we need to clear it to create a new one.

Comment (sdtsui) [reply ui/app/send-v2.js]: @danjm - yep, I like it.  Feel free to add it to next PR.

**Pull request merged**